src/components/icons/index.js

```diff
@@ -0,0 +1,27 @@
+import LabelIcon from '../icons/label';
+import LayersIcon from '../icons/layers';
+import MenuIcon from '../icons/menu';
+import ThemeIcon from '../icons/theme';
+import ExportIcon from '../icons/export';
+import MapIcon from '../icons/map';
+import PlusIcon from '../icons/plus';
+import MinusIcon from '../icons/minus';
+import ResetIcon from '../icons/reset';
+import IndicatorIcon from '../icons/indicator';
+import VisibleIcon from '../icons/visible';
+import InvisibleIcon from '../icons/invisible';
+
+export default {
+  label: LabelIcon,
+  layers: LayersIcon,
+  menu: MenuIcon,
+  theme: ThemeIcon,
+  export: ExportIcon,
+  map: MapIcon,
+  plus: PlusIcon,
+  minus: MinusIcon,
+  reset: ResetIcon,
+  indicator: IndicatorIcon,
+  visible: VisibleIcon,
+  invisible: InvisibleIcon
+};
```

Comment:
I would consider using non-default exports instead, so that you could import individual icons,
e.g.
```es6
import { download } from '../icons';
```
Note that I had to rename 'export' to 'download' because `export` is a reserved keyword.

```suggestion
export const label = LabelIcon;
export const layers = LayersIcon;
export const menu = MenuIcon;
export const theme = ThemeIcon;
export const download = ExportIcon;
export const map = MapIcon;
export const plus = PlusIcon;
export const minus = MinusIcon;
export const reset = ResetIcon;
export const indicator = IndicatorIcon;
export const visible = VisibleIcon;
export const invisible = InvisibleIcon;
```

Reply:
Would that still work as a string id mapping object though as that was the reason for this? Maybe?

Reply:
I think so, e.g.
```es6
import * as icons from '../icons';

const MapIcon = icons['map'];
```

Reply:
Since they are string references, I don't think the bundler would be able to tree shake them though as it will be determined at run-time, though we can instead switch to passing the actual icon components in the prop rather than strings if you like?

Reply:
All icons are now passed as components instead of strings, seems to work well.